### ISSUE TITLE
Uppföljningsmejl en vecka efter två personer blivit ihopkopplade

### ISSUE THREAD

**Issue by jongotlin** [jongotlin/Kompisbyran#21]
För att få bättre feedback på hur matchningen fungerade och för att samla in kommentarer om mötet behövs ett mejl skickas en vecka senare med uppmaning att skriva lite om träffen. Det bör även finnas ett val att kryssa i om träffen inte blev av.

Användaren ska inte behöva logga in för att skriva kommentaren utan en länk med en token räcker. Alternativt lägga till https://github.com/jmikola/JmikolaAutoLoginBundle.

**Comment by EmmaHammar** [comment]
Uppföljningsmail till personer som vill hjälpa någon att öva svenska:
Ämne:  Uppföljning från Kompisbyrån

Hej!

Tack för att du har anmält dig till en kompisfika genom Kompisbyrån!
Vi skulle väldigt gärna vilja veta vad du tycker om din kompisfika för att vi ska kunna förbättra vår verksamhet. Det tar bara några minuter och du svarar anonymt. Du hittar frågorna här: http://goo.gl/forms/k5QyWTPc1g 

Vill du träffa fler fikakompisar som är nya i Sverige? Logga in med din e-post och ditt lösenord här: http://kompisbyran.se/login och anmäl dig till en ny kompisfika.

Stort tack för hjälpen, vi uppskattar det väldigt mycket. 

Bästa hälsningar
Alla på Kompisbyrån 





Uppföljningsmail till personer som vill bli bättre på svenska:
Ämne: Uppföljning från Kompisbyrån (Follow-up from Kompisbyrån)

(This message will follow in English)

Hej!

Tack för att du är med i Kompisbyrån!
Vi vill gärna veta vad du tycker om Kompisbyrån för att bli bättre på vårt jobb. Det tar bara några minuter och du svarar anonymt. Du går in här: http://goo.gl/forms/76v5MgEG9F och svarar på några frågor.

Vi vet många svenskar som vill träffa nya kompisar från andra länder. Om du vill träffa nya fikakompisar, logga in med din e-mail och ditt lösenord här: http://kompisbyran.se/login. Tryck på "Kompisfika"-knappen. Nu är du anmäld igen!

Kontakta oss om du har frågor! 

Bästa hälsningar
Alla på Kompisbyrån 

--------------------------------------------------------------
(Translation in English)

Hello!

Thank you for being a part of Kompisbyrån. We would like to know your thoughts about your meeting with your fika friend in order to improve our organization. It only takes a couple of minutes and you respond anonymously. Go to: http://goo.gl/forms/76v5MgEG9F and answer a few questions. 

We know many Swedes who want to meet new friends from other countries. If you want to meet more new friends, log in with your e-mail and password here: http://kompisbyran.se/login. Press the "Kompisfika"-button. Now, you have signed up again!

Please, contact us if you have any questions. 

Best regards,
Kompisbyrån



![uppfoljning etablerade svenskar_2](https://cloud.githubusercontent.com/assets/16022299/11853209/ec245978-a43c-11e5-962e-44b4ff022dc5.png)
![uppfoljning etablerade svenskar_1](https://cloud.githubusercontent.com/assets/16022299/11853208/ec21d1bc-a43c-11e5-8d29-f70d931b6793.png)
![uppfoljning personer som vill ova 2](https://cloud.githubusercontent.com/assets/16022299/11853210/ec25f918-a43c-11e5-88e6-45867c723c05.png)
![uppfoljning personer som vill ova_1](https://cloud.githubusercontent.com/assets/16022299/11853207/ec21bff6-a43c-11e5-83c1-9d2ad5502703.png)

**Comment by dkvk** [comment]
Är det ett kriterium att det ska vara en unik länk?

Ett alternativ är att man lägger upp alla som anmält sig på en lista hos ex Mailchimp som i sin tur skickar ut mailet efter 3 veckor.

**Comment by EmmaHammar** [comment]
Nej, i dagsläget är det inte en "personlig" länk, så mailchimp borde funka
för det, tt det tre v efter själva matchningen görs skickas ut, vore
verkligen super!

lördag 30 januari 2016 skrev Daniel Kolvik <notifications@github.com>:

**Comment by jongotlin** [comment]
@dkvk, vad är fördelen med att köra genom Mailchimp? Jag tänker annars ett `Command` som körs en gång per dag och tar fram kopplingar som är gjorda för 3v sedan och skickar mejl till dessa.
